docs/proposals/job.md

```diff
@@ -0,0 +1,135 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pod(s) that require to run on demand.
+
+Several existing issues and PRs were already created regarding that particular subject:
+* Job Controller [#1624](https://github.com/GoogleCloudPlatform/kubernetes/issues/1624)
+* New Job resource [#7380](https://github.com/GoogleCloudPlatform/kubernetes/pull/7380)
+
+
+## Use Cases
+1. Be able to start of one or several pods tracked as a single entity.
+1. Be able to implement heavy-computation model, eg. MapReduce.
```

Comment:
> In addition to the features listed, users also want parallelism control (max and min simultaneous instances), task assignment, and queuing. However, I'd start simple.

I was thinking about just the amount of simultaneous instances, without max and mins, actually. Something similar to what we have in `ReplicationController`. Doing max and mins adds unnecessary level of complication imho.